src/extension.ts

```diff
@@ -30,9 +31,9 @@ export function activate(context: vscode.ExtensionContext) {
     const outputChannel: vscode.OutputChannel = vscode.window.createOutputChannel(Constants.edgeDisplayName);
     Simulator.validateSimulatorUpdated(outputChannel);
     const edgeManager = new EdgeManager(context);
-
     const simulator = new Simulator(context);
     const containerManager = new ContainerManager(simulator);
+    const marketplace = new Marketplace(context);
```

Comment:
When will new the marketplace? On demand or persist an instance in context?

Reply:
Good catch. I will remove it. It is unused.